app/assets/stylesheets/govuk_publishing_components/components/govspeak/_highlight-answer.scss

```diff
@@ -43,5 +43,16 @@ $highlight-answer-color: govuk-colour("white");
         }
       }
     }
+
+    // stylelint-disable declaration-no-important
+    @include govuk-media-query($media-type: print) {
+      background-color: unset;
```

Comment:
Just interested - the background colour is already disabled, I'd guess by the browser but I'm not certain. Is there a benefit to explicitly unsetting it as well?

Reply:
If "Print background graphics" is ticked in the printer dialogue settings, the green background will continue to print (looking dark grey on a b&w printer, with black text over it). By explicitly unsetting the background colour here, we override that printer setting and ensure the output will be black on white. 

We could use `none` or `white` or `transparent` or even `initial` values here, but I find the `unset` keyword better communicates the intent (it behaves the same as `initial` in this context).